config/config/common-config.yaml

```
@@ -757,3 +757,6 @@ networkPolicies:
 
   dnsAutoscaler:
     enabled: true
+
+  alerts:
+    enabled: false
```

I think we can have it enabled by default

```suggestion
    enabled: true
```